src/config/cache.config.ts

```diff
@@ -1,14 +1,18 @@
 import { EWC_CHAIN_ID, VOLTA_CHAIN_ID } from '@energyweb/credential-governance';
 import { CacheServerClientOptions } from '../modules/cache-client/cache-client.types';
 
-const cacheConfig: Record<number, CacheServerClientOptions> = {
+const defaultConfig: Record<number, CacheServerClientOptions> = {
   [VOLTA_CHAIN_ID]: {
     url: 'https://identitycache-staging.energyweb.org/v1/',
```

Comment:
If default domain is `switchboard-dev`, we should change `url` to `identitycache-dev`. I think this is actually a good time to do this as this is already a breaking change due to support for the previous login method being removed.

```suggestion
    url: 'https://identitycache-dev.energyweb.org/v1/',
```

Reply:
@JGiter could you also note the above in the migration guide documentation please? If folks had previously relied on the default value, they will need to specify `'https://identitycache-staging.energyweb.org/v1/' as the default has been changed to 'https://identitycache-dev.energyweb.org/v1/'

Reply:
Good idea